src/applications/accreditation/21a/tests/fixtures/data/test-data.json

```diff
@@ -11,10 +11,8 @@
     "dateOfBirth": "1995-01-09",
     "placeOfBirthAddress": {
       "country": "USA",
-      "street": "1234 Main St",
       "city": "City",
-      "state": "CA",
-      "postalCode": "12345"
```

Comment:
What's the reason for removing some of these address fields?

Reply:
place of birth doesn't need all the fine grained address details, just city/state/country